### PR TITLE
Update Roadmap-and-Backlog.md

### PR DESCRIPTION
**Pull request by lfrohlich**
### Summary of Changes
Updates roadmap description to reflect latest roadmap (8/20/2021)

### CODE REVIEW

File: docs/Product-Strategy/Roadmap-and-Backlog.md [hunk 1]
```diff
@@ -45,10 +41,11 @@ _Risks:_ The users that would be on-boarded to this process, would potentially h
 | Files successfully transferred to ACF server | Not Started
 | Files uploaded by STT user(s) are successfully entered into the existing TDRS system | Not Started
 
-## Release 3 (fka STT MVP)
-Value Delivered: Create a database that will parse data, communicate error messages to the user that are easier to read and act upon. The scope of this release is still being defined, but the team agrees that the priority for focusing on processing data is the next right step.
+## Release 3: Parity, Data Parsing, Automated Status, Notifications
+Value Delivered: Create a database that will parse data, communicate error messages to the user that are easier to read and act upon. This release will allow us to onboard users and eventually depreciate the TANF Data Reporting System.
```
Comment (abottoms-coder) [inline docs/Product-Strategy/Roadmap-and-Backlog.md:45]: Just a terminology tweak here: I'd say "Create a software system with a relational database that will...."

File: docs/Product-Strategy/Roadmap-and-Backlog.md [hunk 2]
```diff
@@ -14,29 +14,25 @@ Value Delivered: Get approval for the authority to operate and create a producti
...
-| Users with appropriate privileges can download files that were previously uploaded     | In Review
+| Users with appropriate privileges can download files that were previously uploaded     | Complete
 |Create production environment | In Progress
 
-## Release 1 (fka OFA MVP) Scope
+## Release 1: Secure access and upload to TDP
 Value Delivered: ACF employees can use a secure method to log into the system; this would require a PIV/CAC card.
 
-Our first release to production will include the functionality built for ATO (above) and also include a secure way for ACF employees to access the system, we're currently looking into integrating with HHS AMS/ACF AMS, NextGenXMS or additional options within login.gov. It is important these measures are put into place before sensitive production data is uploaded to the system.
+Our first release to production will include the functionality built for ATO (above) and also include secure ways for users to access the system via ACF AMS for ACF users and NextGen XMS/Login.gov for non-ACF users. It is important these measures are put into place before sensitive production data is uploaded to the system.
```
Comment (ADPennington) [inline docs/Product-Strategy/Roadmap-and-Backlog.md:23]:
```suggestion
Our first release to production will include the functionality built for ATO (above) and also include secure ways for users to access the system via ACF AMS for ACF users and NextGen XMS/Login.gov for non-ACF users. It is important that these measures are put into place before sensitive production data is uploaded to the system.
```

File: docs/Product-Strategy/Roadmap-and-Backlog.md [hunk 3]
```diff
@@ -14,29 +14,25 @@ Value Delivered: Get approval for the authority to operate and create a producti
...
-
-_Notes:_ We're investigating a way to automatically update file names once uploaded (based on standard naming convention) and set up a CRON job to pick up files and move them into a place where the existing TDRS application can pick them up. If this solution is selected, it would reduce the amount of time OFA tribal TANF and data staff spend with each file. 
-
-While this type of workflow isn't our long term workflow, including it at this point delivers value to the users quicker. So we are evaluating the ROI before deciding on this path.
+## Release 2: Early Secure Release
+Value Delivered: Allow users to securely upload data into our system, replacing a less secure way of doing so while increasing communication with the users and not increasing burden for OFA Admin staff. This replease will include a workflow that 8 Tribes can pilot the use of login, upload and download of files. While this type of workflow isn't our long term workflow, including it at this point delivers value to the users quicker.
```
Comment (ADPennington) [inline docs/Product-Strategy/Roadmap-and-Backlog.md:35]:
```suggestion
Value Delivered: Allow users to securely upload data into our system, replacing a less secure way of doing so while increasing communication with the users and not increasing burden for OFA Admin staff. This release will include a workflow for 8 Tribes to pilot the use of login, upload and download of files. While this type of workflow isn't our long term workflow, including it at this point delivers value to the users quicker. 
```

File: docs/Product-Strategy/Roadmap-and-Backlog.md (hunk 1 above)
Comment (ADPennington) [inline docs/Product-Strategy/Roadmap-and-Backlog.md:45]:
```suggestion
Value Delivered: Create a software system with a relational database that will parse data, communicate error messages to the user that are easier to read and act upon. This release will allow us to onboard more users and eventually deprecate the TANF Data Reporting System.
```

**Review by ADPennington — COMMENTED** [review]
lgtm @lfrohlich! made a few minor suggestions, including one from @abottoms-coder 😄

**Review by ADPennington — APPROVED** [review]
👍🏾

**Pull request merged**